Library/Homebrew/utils/curl.rb

```diff
@@ -118,18 +118,10 @@ def curl_download(*args, to: nil, partial: true, **options)
            destination.size == %r{^.*Content-Range: bytes \d+-\d+/(\d+)\r\n.*$}m.match(headers)&.[](1)&.to_i
           return # We've already downloaded all the bytes
         end
-      else
-        supports_partial_download = false
```

Comment:
Is this not accurate? If I understand correctly, the problem is that when this is `false`, `--continue-at 0` will still fail for some web servers. Can we remove only `--continue-at 0` and keep `--continue-at -` then?